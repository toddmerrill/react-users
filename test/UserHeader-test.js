import React from 'react'
import { UserHeader } from '../src/components/Users.js'
import renderer from 'react-test-renderer'
import mock from './user-mocks'

it('UserHeader renders correctly', () => {
    const tree = renderer.create(
        <UserHeader users={mock.users} currentUser={mock.user} addUser={mock.f} removeUser={mock.f}/>
    ).toJSON();
    expect(tree).toMatchSnapshot();
});
