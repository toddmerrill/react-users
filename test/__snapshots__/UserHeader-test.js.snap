// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserHeader renders correctly 1`] = `
<div
  className="listHeader"
>
  <h2
    className="oneLine"
  >
    Users
  </h2>
  <span
    className="rightButtons"
  >
    <a
      className="nohover addButton"
      onClick={[Function]}
    >
      Add User
    </a>
    <a
      className="nohover deleteButton"
      onClick={[Function]}
    >
      Delete User
    </a>
  </span>
</div>
`;
